### ISSUE TITLE
Remove Nightwatch from `vets-website`

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#26560]

**Comment by CBonade** [comment]
https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/27067

The above issue is tracking specs that have yet to be converted to Cypress.  Once converted and tested for a period of time, the nightwatch specs can be removed and then this deprecation process can continue.

**Comment by CBonade** [comment]
Spec conversion is ongoing around other tasks

**Comment by CBonade** [comment]
Progress has been made in the associated ticket listed above.

**Comment by CBonade** [comment]
Associated ticket listed above has more progress, only 3 specs outstanding.  @pjhill initially this ticket started out with the intention of just removing Nightwatch and any associated config/helper files.  As all the other specs were discovered (unfortunately after much of Nightwatch's config had been deleted), the rewrites have merged into this branch.  Should I assume we will go through the same process of pushing these new specs first, letting them sit for 10 days and then going back and removing everything else?  If so, I'll roll reverting the nightwatch deletion into the tasks that need to be done here.

**Comment by CBonade** [comment]
Nightwatch was added back in until the new tests spend a bit of time in the field and show dependability.  Will be a few days at least before more progress can be made here.

**Comment by CBonade** [comment]
Awaiting two reviews to merge in the first batch of specs here.  Still investigating an alternate solution for cross-origin testing in cypress

**Comment by CBonade** [comment]
Still awaiting the two reviews from yesterday (will directly request from the teams tomorrow).  The user nav cross-origin issue has a potential alternative worked out where we verify the error messages are as we would expect them to be.  Keyboard tests are still awaiting in-development helpers.

**Comment by CBonade** [comment]
User nav test has been solved.  Main bulk of the refactors are still awaiting 4 more reviews before we can move into our "monitoring" phase.

**Comment by CBonade** [comment]
No additional reviews today - still awaiting 4 teams.

**Comment by CBonade** [comment]
No new reviews, awaiting 4 more teams.

**Comment by pjhill** [comment]
Joe brought up a good point, namely that we will need to tweak the Jenkinsfile to ensure that Jenkins workflows continue to run without error now that Nightwatch specs and the supporting Nightwatch infrastructure have been removed from `vets-website`.

I don't think you have a PR or draft PR yet, but I expect that we will be removing the scripts responsible for executing Nightwatch. If Jenkins attempts to execute those now missing scripts then I'm not sure how it will fail out of the workflow.

We should probably remove the Nightwatch execution for now from Jenkinsfile [here ](https://github.com/department-of-veterans-affairs/vets-website/blob/40c24474e99f9a1114add503183773fe8bba9fef/Jenkinsfile#L129)and [here](https://github.com/department-of-veterans-affairs/vets-website/blob/40c24474e99f9a1114add503183773fe8bba9fef/Jenkinsfile#L155).

**Comment by CBonade** [comment]
In my branch that has Nightwatch being removed, the Jenkinsfile and GHA workflows have both had Nightwatch removed.

That said, my branch is resulting in unit tests failing which seems odd to me as the unit tests have nothing to do with Nightwatch. Currently investigating and troubleshooting.

**Comment by CBonade** [comment]
Nightwatch is no longer running in our GHA or Jenkins workflows. The process is now under way of alerting teams to their specs being removed for approval, and the last step will be removing global configs and helpers.

**Comment by CBonade** [comment]
Happy to report this is finally complete - Nightwatch, it's old specs, configs and helpers are gone from vets-website.

**Comment by pjhill** [comment]
Hooray -- this is awesome!
